pkg/job/scrape.go

```diff
@@ -38,6 +39,12 @@ func ScrapeAwsData(
 	awsInfoData := make([]model.TaggedResourceResult, 0)
 	var wg sync.WaitGroup
 
+	enhancedProcessor, err := enhancedmetrics.NewProcessor(factory)
+	if err != nil {
+		logger.Debug("Couldn't initialize enhanced metrics processor", "err", err)
```

Comment:
Do you think this should be a Warning level message? Or is it expected that this fails in most cases?